s3/pkg/service.go

```diff
@@ -48,6 +48,23 @@ var SupportedClasses []pb.StorageClass
 
 type s3Service struct{}
 
+func getTierFromName(backendType, className string) (int32, S3Error) {
+	v, ok := Ext2IntTierMap[backendType]
```

Comment:
this map is a local map ? how will we ensure this is in sync with AWS, Azure etc. always ? Is it better we call their API to do the mapping always ?

Reply:
This is a map between tier used in OpenSDS to storage class in cloud platform, no API of cloud platform used for this, you can see it from the design spec[https://github.com/opensds/design-specs/blob/master/specs/capri/object_lifecycle_management.md](url).